api-tests/api-audit/src/test/java/endpoints/codeReview/peerReview.feature

```diff
@@ -0,0 +1,14 @@
+Feature: peerReview
+
+  Background:
+    * header apiUser = 'apiadmin'
+    * header Authorization = call read('classpath:basic-auth.js')
+    * url baseUrl
+    * configure readTimeout = 60000
+
+  Scenario: Verify pull request information for a repo
```

Comment:
We have more audit statuses so please add a //ToDo indicating that this is just the basic setup check
and a pointer to where those statuses are documented - for example the readme in api-audit